esp/esp/web/util/latex.py

```diff
@@ -46,6 +47,7 @@
 
 TEX_TEMP = tempfile.gettempdir()
 TEX_EXT  = '.tex'
+devnull_sentinel = object()
```

Comment:
If importing this from elsewhere is supported, I'd just name it `devnull`, since the fact that it's a sentinel object is an implementation detail.  If it's not (which would support), it should start with an underscore.